index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html xml:lang="en" lang="en">
+
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+  <title>Portfolio Project</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;700;800&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
+  <link rel="stylesheet" type="text/css" href="style.css" />
+</head>
+
+<body>
+
+  <div class="header">
+    <img class="logo" src="My Logo.png">
+    <nav class="flex-nav">
+      <a href="#" class="toggleNav"><i class="fa fa-hamburguer"></i>= Menu</a>
+      <ul class="open">
+        <li><a href="#">Portfolio</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+        </ul>
+    </nav>
+  </div>
```

Comment:
- [ ] For semantics reasons Please ensure adding the `<header>` section and wrap up all the corresponding content inside it 💪